playbooks/openstack-caasp_instance.yml

```diff
@@ -10,6 +10,9 @@
     - name: Load openstack vars
       include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
 
+    - name: Preflight OpenStack Caasp resources
+      include: _openstack_preflight_caasp.yml
```

Comment:
This gets now called whenever we call the deploy caasp step. Even if the stack got already successfully deployed. I don't think that is needed and wasted time

Reply:
refactored